docs/source/actions.rst

```diff
@@ -196,13 +196,19 @@ List of recipients of the message. It could be a channel or an user.
 
 SPIDERMON_SLACK_SENDER_NAME
 ---------------------------
+Name of app/bot/user created for serving notifications on desired channels.
 
 .. _SPIDERMON_SLACK_SENDER_TOKEN:
 
 SPIDERMON_SLACK_SENDER_TOKEN
 ----------------------------
 
-Your `Slack token`_.
+Read `here`_ to know more about Slack Tokens.
+Get your Slack Credentials by `creating an app <https://api.slack.com/apps>`_  here.
+
+The `SPIDERMON_SLACK_SENDER_TOKEN` is the `API Token` available in your Slack application dashboard. You can navigate to Browse apps> Custom Integrations > Bots > Edit Configuration. 
+
+Warning: Be careful when using bot user tokens in Spidermon. Do not publish bot user tokens in public code repositories.
```

Comment:
Is this really specific about bots? Isn’t this something that applies to any Slack token?

On an :lipstick: aesthetic side: use the [`warning` directive](http://www.sphinx-doc.org/es/stable/markup/para.html#directive-warning).

Reply:
Will correct that.